src/Engine/babylon.engine.ts

```diff
@@ -3988,52 +3988,60 @@
         }
 
         public createRenderTargetCubeTexture(size: number, options?: RenderTargetCreationOptions): InternalTexture {
-            var gl = this._gl;
-
-            var texture = new InternalTexture(this, InternalTexture.DATASOURCE_RENDERTARGET);
-
-            var generateMipMaps = true;
-            var generateDepthBuffer = true;
-            var generateStencilBuffer = false;
-
-            var samplingMode = Texture.TRILINEAR_SAMPLINGMODE;
-            if (options !== undefined) {
-                generateMipMaps = options.generateMipMaps === undefined ? true : options.generateMipMaps;
-                generateDepthBuffer = options.generateDepthBuffer === undefined ? true : options.generateDepthBuffer;
-                generateStencilBuffer = (generateDepthBuffer && options.generateStencilBuffer) ? true : false;
+            let fullOptions = new RenderTargetCreationOptions();

```

Comment:
Hello, I suggest using partial here:
https://github.com/BabylonJS/Babylon.js/blob/master/src/Helpers/babylon.environmentHelper.ts#L317

Reply:
I don't know how to use partial (I don't use typescript actually...) but I can add it blindly. :)

Reply:
Not too blindly :D please compile and test before

Reply:
Of course haha